cmd/account/transfer.go

```diff
@@ -0,0 +1,101 @@
+package account
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+
+	staking "github.com/oasisprotocol/oasis-core/go/staking/api"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/connection"
+	sdkSignature "github.com/oasisprotocol/oasis-sdk/client-sdk/go/crypto/signature"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/helpers"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/modules/accounts"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/types"
+
+	"github.com/oasisprotocol/cli/cmd/common"
+	cliConfig "github.com/oasisprotocol/cli/config"
+)
+
+var transferCmd = &cobra.Command{
+	Use:   "transfer <amount> <to>",
+	Short: "Transfer given amount of tokens to a different account",
+	Args:  cobra.ExactArgs(2),
+	Run: func(cmd *cobra.Command, args []string) {
+		cfg := cliConfig.Global()
+		npa := common.GetNPASelection(cfg)
+		txCfg := common.GetTransactionConfig()
+		amount, to := args[0], args[1]
+
+		if npa.Account == nil {
+			cobra.CheckErr("no accounts configured in your wallet")
+		}
+
+		// When not in offline mode, connect to the given network endpoint.
+		ctx := context.Background()
+		var conn connection.Connection
+		if !txCfg.Offline {
+			var err error
+			conn, err = connection.Connect(ctx, npa.Network)
+			cobra.CheckErr(err)
+		}
+
+		// Resolve destination address.
+		toAddr, toEthAddr, err := common.ResolveLocalAccountOrAddress(npa.Network, to)
+		cobra.CheckErr(err)
+
+		// Check, if to address is known to be unspendable.
+		common.CheckForceErr(common.CheckAddressNotReserved(cfg, toAddr.String()))
+
+		acc := common.LoadAccount(cfg, npa.AccountName)
+
+		var sigTx, meta interface{}
+		switch npa.ParaTime {
+		case nil:
+			if txCfg.Encrypted {
+				cobra.CheckErr("--encrypted not supported for consensus transactions")
+			}
```

Comment:
You could make this general and put the check into `SignConsensusTransaction` so it is always performed?